app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -402,6 +409,8 @@ function toField(
     headerTooltip: displayValue ? displayValue : '',
     tooltipComponentParams: {
       numberOfNothing: index != null ? dataQuality.number_of_nothing[index] : null,
+      numberOfLeadOrTrailSpace:
+        index != null ? dataQuality.number_of_leading_trailing_whitespace[index] : null,
       numberOfWhitespace: index != null ? dataQuality.number_of_whitespace[index] : null,
```

Comment:
If these values are computed earlier in the function, they could be used to simplify the `showDataQuality` logic.